eos_data_distribution/avahi_monitor.py

```diff
@@ -59,13 +68,21 @@ class AvahiMonitor(object):
     def __init__(self):
         super(AvahiMonitor, self).__init__()
 
+        self.ips = []
```

Comment:
This block of private variable declarations could do with some documentation on what they all mean and how they interact.